src/server/dfly_main.cc

```diff
@@ -18,6 +11,13 @@
 #include <absl/strings/str_split.h>
 #include <absl/strings/strip.h>
 
+#include "absl/cleanup/cleanup.h"
+#include "absl/container/inlined_vector.h"
+#include "absl/strings/numbers.h"
+
+#define INJECT_ALLOCATION_TRACKER
```

Comment:
Can you please introduce a CMakeLists option `DFLY_MEMORY_TRACKING` in src/CMakelists.txt ? 
Let's set it to ON by default and it will add the conditional define to dfly_main.cc. 
In case it's disable we fallback to `#include <mimalloc-new-delete.h>`

Reply:
ah, I see you add this define to every file but use it only here. ok